docs/index.md

```diff
@@ -2,16 +2,27 @@
 
 Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-Currently, Pulse supports a push-based integration with your workflow and the recommended way of sending the necessary data to Pulse is by using the Pulse CLI.
+Currently, Pulse supports a push-based integration with your workflow and the recommended way of reporting the necessary events to Pulse is by using the Pulse CLI.
 
-However, in some scenarios it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are sending data from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
+## Before you begin
 
-`https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>`
+Consider the following before integrating your workflow with Pulse:
 
-!!! important
-    **Please let us know if you are planning on pushing events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks. For these situations, you can call an HTTP POST webhook instead. For example:
```

Comment:
Not sure why we are adding this again in the docs but this is probably as bad idea since we will receive data that will not be processed.

Reply:
This information about the HTTP endpoint has been in the public documentation "since the beginning" :sweat_smile:, here I just reformatted it inside an unordered list. I'm keeping the admonition asking customers to let us know if they need to use the endpoint - this can provide us with data about customers that need this capability to integrate with Pulse.